arcgis-ios-sdk-samples/Search/Find place/FindPlaceViewController.swift

```diff
@@ -145,16 +140,14 @@ class FindPlaceViewController: UIViewController {
         }
     }
     
-    // method to clear prefered location information
-    // hide the suggestions table view, empty previously selected
-    // suggest result and previously fetch search location
+    /// Clear prefered location information, hide the suggestions table view, empty previously selected suggest result and previously fetch search. location
```

Comment:
```suggestion
    /// Clear preferred location information, hide the suggestions table view, empty previously selected suggest result and previously fetch search. location
```

Also it would be good if the comment can fit under the 80-col ruler/width guide.